WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -62,6 +62,13 @@ public WabiSabiConfig(string filePath) : base(filePath)
 		[JsonProperty(PropertyName = "RegistrableWeightCredentials", DefaultValueHandling = DefaultValueHandling.Populate)]
 		public uint RegistrableWeightCredentials { get; set; } = 1_000;
 
+		/// <summary>
+		/// Calculation can be found here https://github.com/zkSNACKs/WalletWasabi/issues/5438.
+		/// </summary>
+		[DefaultValue(399834)]
+		[JsonProperty(PropertyName = "MaxCoinJoinTransactionWeight", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public uint MaxCoinJoinTransactionWeight { get; set; } = 399834;
```

Comment:
I made two mistakes in the calculation:

- forgot to account for *both* `txins_count` and `txouts_count`
- i must have made some other arithmetic error, because `399834 - (400000 - ( 4 * (8 + 6 + 31)  + 2 ) )` is 16, not 12 as expected

i corrected the issue, so the reference in the doc comment is up to date

```suggestion
		[DefaultValue(399818)]
		[JsonProperty(PropertyName = "MaxCoinJoinTransactionWeight", DefaultValueHandling = DefaultValueHandling.Populate)]
		public uint MaxCoinJoinTransactionWeight { get; set; } = 399818;
```

Reply:
ACK